rust-toolchain

```diff
@@ -1 +1 @@
-nightly-2019-07-08
+beta-2019-09-25
```

Comment:
Can we now just pin to `beta` instead of a specific one?

Reply:
Yes we could, let me try that out